include/libwebsockets/lws-context-vhost.h

```diff
@@ -1040,6 +1040,16 @@ enum lws_mount_protocols {
 	LWSMPRO_CALLBACK	= 6, /**< hand by named protocol's callback */
 };
 
+/** enum lws_authentication_mode
+ * This specifies the authentication mode of the mount. For compatibility reasons, if
+ * basic_auth_login_file is non-null then basic auth file
+ */
+enum lws_authentication_mode {
+	LWSAUTHM_DEFAULT = 0, /**< default authenticate only if basic_auth_login_file is provided */
+	LWSAUTHM_BASIC_AUTH_CALLBACK = 1 /**< Basic auth with a custom verifier */
```

Comment:
You don't need to take care about =0 or =1, enum is specified to start at 0 and add one each time.

Reply:
Yeah, I was just copying the code right above for `lws_mount_protocols` to be consistent. I am fine either way